napari/layers/_shapes_layer/shape_list.py

```diff
@@ -668,6 +681,12 @@ def to_labels(self, labels_shape=None, shape_type=None):
         labels_shape : np.ndarray | tuple | None
             2-tuple defining shape of labels image to be generated. If non
             specified, takes the max of all the vertiecs
+        zoom_factor : float
+            Premultiplier applied to coordinates before generating mask. Used
+            for generating as downsampled mask.
+        offset : 2-tuple
+            Offset preapplied to coordinates before multiplying by the
+            zoom_factor. Used for putting negative coordinates into the mask.
```

Comment:
(a) used how?, and (b) what happens if there are negative coordinates and offset remains [0, 0]?

Reply:
I've changed this to `added`, and added `np.clip` to the code, and changed the doc string to reflect this

Reply:
I meant `subtracted`!! have fixed :-)